src/AutoRest.CSharp/Mgmt/Decorator/Transformer/SchemaFormatByNameTransformer.cs

```diff
@@ -43,7 +43,7 @@ internal static void Update()
         private IEnumerable<Schema> allGeneralSchemas;
         private IEnumerable<OperationGroup> allOperationGroups;
         private IReadOnlyDictionary<string, string> allFormatByNameRules;
-        private Dictionary<Guid, string> schemaCache = new Dictionary<Guid, string>();
+        private Dictionary<Schema, string> schemaCache = new();
```

Comment:
Just for curiosity, why is this change needed?

Reply:
1. If we have to introduce an Id in schema, we should not do that in the CodeModel.cs file. We should put it in CodeModelPartial.
2. This has no point since the instance itself works as an unique Id of a schema object. We are doing that a lot in MgmtOutputLibrary, quite a few dictionaries are using the objects from code model as keys (such as Schema, Operation, etc)